meson.options

```diff
@@ -0,0 +1,26 @@
+# Build the HTML / PDF documentation
+option('build-docs', type: 'boolean')
+
+# Build sage.graphs.bliss
```

Comment:
Not sure if it's more helpful (but perhaps more future-proof) to say "everything that relies on `bliss`" instead of the specific sage module that currently uses it.

Reply:
Agreed, I tried to make these more sensible now that they're user-facing.